fotoobo/cli/fc/get.py

```diff
@@ -0,0 +1,94 @@
+"""
+The FortiCloud get commands
+"""
+
+import logging
+from pathlib import Path
+from typing import Any, Union
+
+import typer
+
+from fotoobo.exceptions.exceptions import GeneralError
+from fotoobo.helpers import cli_path
+from fotoobo.helpers.files import save_json_file
+from fotoobo.helpers.result import Result
+from fotoobo.tools import fc
+
+app = typer.Typer(no_args_is_help=True, rich_markup_mode="rich")
+log = logging.getLogger("fotoobo")
+
+
+HELP_TEXT_OPTION_OUTPUT_FILE = "The file to write the output to."
+
+
+@app.callback()
+def callback(context: typer.Context) -> None:
+    """
+    The fc get subcommand callback
+
+    Args:
+        context: The context object of the typer app
+    """
+    cli_path.append(str(context.invoked_subcommand))
+    log.debug("About to execute command: '%s'", context.invoked_subcommand)
+
+
+@app.command()
+def products(
+    output_file: Union[None, Path] = typer.Option(
+        None,
+        "--output",
+        "-o",
+        help=HELP_TEXT_OPTION_OUTPUT_FILE,
+        metavar="[output]",
+    ),
+    raw: bool = typer.Option(False, "-r", "--raw", help="Output raw data."),
+) -> None:
+    """
+    Get the FortiCloud products from asset management.
+    """
+    result = fc.get.products("forticloud")
```

Comment:
I think that "forticloud" is hardcoded here is not by intention? I think we should be able to provide the "host" via command line (optionally, like with the others)